src/gmt_init.c

```diff
@@ -354,33 +354,46 @@ static struct GMT_FONTSPEC GMT_standard_fonts[GMT_N_STANDARD_FONTS] = {
 #include "standard_adobe_fonts.h"
 };
 
-/* List of GMT common keyword/options pairs */
-#ifdef USE_GMT_KWD
-struct GMT_KW_DICT gmt_kw_common[] = {
-	{'R', "region"},
-	{'J', "proj"},
-	{'U', "timestamp"},
-	{'V', "verbose"},
-	{'X', "xoff"},
-	{'Y', "yoff"},
-	{'a', "aspatial"},
-	{'b', "binary"},
-	{'d', "nodata"},
-	{'e', "find"},
-	{'f', "coltypes"},
-	{'g', "gap"},
-	{'h', "header"},
-	{'i', "incol"},
-	{'o', "outcol"},
-	{'n', "interpol"},
-	{'p', "perspective"},
-	{'r', "registration"},
-	{'t', "transparency"},
-	{'x', "cores"},
-	{':', "order"},
-	{'\0', ""}	/* End of list marked with empty code and string */
+/* List of GMT common keyword/options pairs.  This list is used in gmtinit_kw_replace to convert
+ * the new long-format GMT options (e.g., --timestamp="My plot"+o5c/6c) to regular GMT short format
+ * options (e.g., -U/5c/6c/"My plot") that the common and module parsers expect.
+ *
+ * For testing this there are two define statements that need to be set in ConfigUser.cmake:
+ *
+ * -DUSE_COMMON_LONG_OPTIONS will allow us to test the gmt_common_kw array below.
+ * -DUSE_MODULE_LONG_OPTIONS will allow us to test any module_kw settings in the modules
+ *
+ * Without these we are blind to the keyword arrays.  Note that while you can test the
+ * common options without the module options, you cannot do the reverse.
+  */
+
+GMT_LOCAL struct GMT_KEYWORD_DICTIONARY gmt_common_kw[] = {
+	/* separator, short-option, long-option, short-directives, long-directives, short-modifiers, long-modifiers */
+	{   0, 'B', "frame",         "",        "",                                        "",         "" },
+	{ '/', 'I', "increment",     "",        "",                                        "e,n",      "exact,number" },
```

Comment:
**-I** isn't a common option.

Reply:
Well, it is and it isn't.  Because it is almost a common option I keep it with the rest in the gmt_common_kw.

Reply:
grdimage also has a **-I** option with a different syntax. Does it cause troubles for grdimage?

Reply:
I think in practice I will need to possibly move that 'I' entry in gmt_common_kw to the local_kw arrays as needed to avoid making it too complicated.  Will need to experiment with that.